app/features/worksheet_generator/document_loaders.py

```diff
@@ -0,0 +1,372 @@
+from langchain_community.document_loaders import YoutubeLoader
```

Comment:
You can import the document loaders in a single line.

Reply:
Done!